lib/capybara_discoball/server.rb

```diff
@@ -0,0 +1,11 @@
+require 'capybara/server'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.